tests/microos/networkmanager.pm

```diff
@@ -108,16 +129,18 @@ true'
     die 'wrong DNS-Manager is currently used for dnsmasq' if ($RcManager !~ /symlink/ || $mode !~ /dnsmasq/);
     ping_check;
     # systemd-resolved
-    assert_script_run('rm -f /etc/NetworkManager/conf.d/00-use-dnsmasq.conf');
-    assert_script_run('ln -rsf /run/systemd/resolve/stub-resolv.conf /etc/resolv.conf');
-    systemctl('restart NetworkManager');
-    record_info('with systemd-resolved');
-    # DNS-Manager check
-    ($RcManager, $mode) = dns_mgr();
-    die 'wrong DNS-Manager is currently used for systemd-resolved' if ($RcManager !~ /unmanaged/ || $mode !~ /systemd-resolved/);
-    ping_check;
-    # Restore the original system config
-    restore_config;
+    unless (is_vmware) {
```

Comment:
@tinawang123 why vmware is skipped here?

Reply:
VMware vm system doesn't have '/run/systemd/resolve/stub-resolv.conf'